ansible/roles/ocp4-workload-dil-streaming/templates/instructions-group.yaml.j2

```diff
@@ -0,0 +1,6 @@
+kind: Group
+apiVersion: user.openshift.io/v1
+metadata:
+  name: '{{ocp4_workload_dil_serverless_webapp_group_name}}'
```

Comment:
Please add space around the `{{` here and a couple of lines down

Reply:
done, also fix workload name